appinfo/info.xml

```diff
@@ -44,6 +44,8 @@ In your Nextcloud instance, simply navigate to **»Apps«**, find the
 	</screenshot>
 	<dependencies>
 		<nextcloud min-version="25" max-version="27" />
+		<lib>ext-pdo</lib>
+		<lib>ext-pdo_sqlite</lib>
```

Comment:
```suggestion
		<lib>pdo</lib>
		<lib>pdo_sqlite</lib>
```

Never seen the ext- prefix used in that context

Reply:
Ok, I was not sure which one is correct. In `package.json` it's `ext-pdo`. Changed now.